script.embuary.helper/addon.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.embuary.helper" name="Embuary Helper" version="2.0.0" provider-name="sualfred">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.module.pil" version="1.1.7"/>
+    </requires>
+    <extension point="xbmc.python.script" library="default.py">
+        <provides>executable</provides>
+    </extension>
+    <extension point="xbmc.python.pluginsource" library="plugin.py">
```

Comment:
I'm not really a fan of having a plugin for a script named an helper. The url `plugin://script.embruary.helper` sounds like an hack. Can't you refactor the plugin part somewhere else?

Reply:
Very common and used by a lot of addons. Maybe I'll split it up into two different addons in the future. But no time for it atm.